pkg/namestore/namestore.go

```diff
@@ -59,6 +63,27 @@ func (x *nameStore) Acquire(name, id string) error {
 		if b, err := os.ReadFile(fileName); err == nil {
 			return fmt.Errorf("name %q is already used by ID %q", name, string(b))
 		}
+		containers, err := client.Containers(ctx)
+		if err != nil {
+			return err
+		}
+		var eg errgroup.Group
+		for _, c := range containers {
+			c := c
+			eg.Go(func() error {
+				info, err := c.Info(ctx, containerd.WithoutRefreshedMetadata)
+				if err != nil {
+					return err
+				}
+				if getContainerName(info.Labels) == name {
+					return fmt.Errorf("name %q is already used by ID %q", name, info.ID)
```

Comment:
Can you please update the error message to ask user to remove the container using `rm -f`